src/storage/storage.cc

```diff
@@ -152,9 +153,14 @@ rocksdb::Options Storage::InitRocksDBOptions() {
       options.compression_per_level[i] = config_->rocks_db.compression;
     }
   }
+
   if (config_->rocks_db.row_cache_size) {
-    options.row_cache = rocksdb::NewLRUCache(config_->rocks_db.row_cache_size * MiB);
+    // Parameters for the rocksdb::LRUCacheOptions here are set in line with the default parameters for the
+    // rocksdb::NewLRUCache function, which is now deprecated
+    rocksdb::LRUCacheOptions lru_cache_options(config_->rocks_db.row_cache_size * MiB, -1, false, 0.5);
```

Comment:
Could we add some named constants for these default arguments (e.g. -1, false, 0.5)?

Reply:
I have did this, but I am not sure the name of my constants are proper.